extension/submitting-a-log.md

```diff
@@ -10,21 +10,23 @@ position: 30
 
 # Submitting a Log
 
-Logs captured by the Fiddler Jam extension can be submitted to the [Fiddler Jam portal]({%slug fj-portal%}), shared with portal users via email, or saved as a [JSON-formatted archive (**HAR**)](https://en.wikipedia.org/wiki/HAR_(file_format)).
+Logs recorded by the Fiddler Jam extension can be submitted to the [Fiddler Jam portal]({%slug fj-portal%}) and shared with portal users via email or a direct link.
```

Comment:
I assume that `via email` refers to "sharing with specific people". However, in this case `via email` carries a more generic meaning. Consider rewording to sound more specific.

```suggestion
Fiddler Jam logs can be submitted to the [Fiddler Jam portal]({%slug fj-portal%}) and shared with portal users via email or a direct link.
```